cilium/cmd/debuginfo.go

```diff
@@ -41,13 +45,26 @@ const (
 	STDOUT outputType = 0 + iota
 	MARKDOWN
 	HTML
+	JSONOUTPUT
+	JSONPATH
```

Comment:
This might be out of scope for this PR but since you're refactoring: Perhaps we can just use the string variables from below as constants directly. The strings are interned anyway so equality checks etc. would work as now (if not better, since no conversion to an int is needed).